remove vertex in a graph

unical1988 opened the pull request.

unical1988 commented:
@yangxk1 sure I will do that shortly.

yangxk1 commented:
@unical1988, Could you please add some unit tests to cover the new functionality?

unical1988 commented:
@yangxk1 sure I will add tests

unical1988 commented:
@yangxk1 I pulled the changes since the last commit and adjusted the code by removing graphProto(),  let me know if the approach to removing edge/vertex is suitable, I will submit the tests after that.

yangxk1 commented:
hi @unical1988  why is the change sooooooo big. 
I suggest you (1) cloese this pr (2)create a new branch from **latest** main branch (3)move the necessary changes to the new branch (4) **re-open** a new pr

unical1988 commented:
> hi @unical1988 why is the change sooooooo big. I suggest you (1) cloese this pr (2)create a new branch from **latest** main branch (3)move the necessary changes to the new branch (4) **re-open** a new pr

It seems the changes of reformatting affected that large number of files, i will move that to another PR
